lib/chef/win32/file.rb

```diff
@@ -155,6 +155,27 @@ def self.verify_links_supported!
         end
       end
 
+      def self.file_access_check(path, desired_access)
+        security_descriptor = Chef::ReservedNames::Win32::Security.get_file_security(path)
+        token_rights = Chef::ReservedNames::Win32::Security::TOKEN_IMPERSONATE | 
```

Comment:
We shouldn't need impersonation right. At most TOKEN_QUERY | STANDARD_RIGHTS_READ I would guess.